deploy/csi-blob-node.yaml

```diff
@@ -34,6 +35,32 @@ spec:
       priorityClassName: system-node-critical
       tolerations:
         - operator: "Exists"
+      initContainers:
+        - name: install-blobfuse-proxy
+          image: mcr.microsoft.com/k8s/csi/blob-csi:latest
+          imagePullPolicy: IfNotPresent
+          command:
+            - "/blobfuse-proxy/init.sh"
+          securityContext:
+            privileged: true
+          env:
+            - name: DEBIAN_FRONTEND
```

Comment:
It's better provide an env or some switch to disable blobfuse-proxy in deploy/*.yaml config, similar to charts config, since this blobfuse-proxy script actually does not work on non-Ubuntu node, wdyt?

Reply:
How about tracking it in another issue? And I will fix it after this PR merged.

Reply:
sure, pls file an issue, thanks.

Reply:
https://github.com/kubernetes-sigs/blob-csi-driver/issues/695